oak/server/module_invocation.cc

```diff
@@ -108,7 +134,7 @@ void ModuleInvocation::SendResponse(bool ok) {
 
     // Restart the gRPC flow with a new ModuleInvocation object for the next request
     // after processing this request.  This ensures that processing is serialized.
-    auto* request = new ModuleInvocation(service_, queue_, node_);
+    auto* request = new ModuleInvocation(service_, queue_, grpc_in_, grpc_out_);
```

Comment:
Not a big fan of the auto* notation. I think it's pretty clear request is a pointer

Reply:
Done.